app/controllers/users_controller.rb

```diff
@@ -70,7 +70,10 @@ def authorized_user
 
   def find_new_badge
     if current_user.mentee?
-      @new_badge = current_user.badges.merge(AssignedBadge.not_displayed).first
+      @new_badge = current_user.undisplayed_badges.first
```

Comment:
are we using somewhere this code?

Reply:
Yes we are... In the dashboard and show views.